### PR TITLE
Use light grey for siteNav text

### LINKED ISSUE #542: siteNav: use lighter color for text
v1.15.2

Current:
image

Suggestions:
* Use slightly lighter grey (compared to main body text) text for menu items so that there is a slight differentiation between siteNav and main text. This seem to be a common practice among the siteNavs I've seen recently.
* Not use blue for expanded menu items as it attracts unnecessary attention to the item. But we can use different styles for different level items if that helps (at least two styles for top-level and sub-levels)

### PR DESCRIPTION
**Pull request by Xenonym**
**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Enhancement to an existing feature

Resolves #542.

**What is the rationale for this request?**
We can have lighter coloured text for the site navigation sidebar to give more relative importance to the main text.

**What changes did you make? (Give an overview)**
Adjust the `color`for `siteNav` text to be `#444444`.

**Testing instructions:**
1. Observe that in MarkBind documents with a navigation sidebar, the colour of the text is `#444444`.

### CODE REVIEW

**Comment by damithc** [comment]
A little bit lighter? See https://hexo.io/docs/ for an example.

**Comment by Xenonym** [comment]
@damithc actually, I based it off Hexo; `#444444` is the `color` of `.sidebar-link`. We could use the built-in Bootstrap `grey`, which is `#6c757d`. Here's what that looks like:

**Comment by damithc** [comment]
@Chng-Zhi-Xuan what do you think?

**Comment by Chng-Zhi-Xuan** [comment]
Looks ok to me. If you would like some accent colour, we can keep the blue on the rotating arrow?

File: asset/css/site-nav.css [hunk 1]
```diff
@@ -91,7 +92,7 @@
 }
 
 .rotate-icon {
-    color: #0076FF;
+    color: #444444;
```
Comment (Chng-Zhi-Xuan) [inline asset/css/site-nav.css]: Let's keep `#0076FF` on the `.rotate-icon`.

Comment (Xenonym) [reply asset/css/site-nav.css]: Sure thing! What do you feel about `#444444` vs `#565656`? I am leaning towards the latter.

Comment (Chng-Zhi-Xuan) [reply asset/css/site-nav.css]: Let's proceed with `#565656`.

**Pull request merged**